include/xkbcommon/xkbcommon-keysyms.h

```diff
@@ -3250,6 +3250,11 @@ performance, or use of this material.
 #define XKB_KEY_longminus            0x100000f6
 #define XKB_KEY_block                0x100000fc
 
+/***********************************************************
+ * Keysyms are 32-bit integers with the 3 most significant bits always set
+ * to zero. See: Appendix A “KEYSYM Encoding” of the X Window System Protocol.
+ */
+#define XKB_MAX_KEYSYM               0x1fffffff
```

Comment:
This define should go in `xkbcommon.h`, like the other MAX defines. For consistency, it should be called `XKB_KEYSYM_MAX`.

Reply:
Done